packages/runtime-common/indexer.ts

```diff
@@ -339,6 +344,33 @@ export class Indexer {
     ]);
   }
 
+  private rangeCondition(filter: RangeFilter, on: CodeRef): CardExpression {
+    on = filter.on ?? on;
+    return every([
+      this.typeCondition(on),
+      ...flatten(
+        Object.entries(filter.range).map(([path, range]) => {
+          let query = fieldQuery(path, on, false, 'filter');
```

Comment:
Let’s follow the same patten as the likeCondition and eqCondition and split this inside of this Object.entries into a different method: fieldRangeFilter. Id like to have this line up with those other two filters closely in their structure.

Reply:
Updated [650053e](https://github.com/cardstack/boxel/pull/1194/commits/650053ed178b2c2bca48f9c125b81fadebfeb010)